src/App.jsx

```diff
@@ -1,21 +1,32 @@
 import { useState, useEffect } from 'react'
 import './App.css'
+import {Routes, Route} from "react-router-dom"
 import Header from './components/Header'
 import Articles from './components/Articles'
+import Article from './components/Article'
 import Searchbar from './components/Searchbar'
-import {Routes, Route} from "react-router-dom"
 
 
 function App() {
-
   const [articles, setArticles] = useState([])
 
   return (
+    <>
```

Comment:
You already have a div as a parent element, there is no need for a fragment here. Could the div be a more semantic element??